contrib/mac/app/Makefile

```diff
@@ -36,7 +36,7 @@ dmg/Applications:
 dmg/$(APP_NAME): startup.applescript julia.icns
```

Comment:
I just noticed also this file below does the same codesigning as the new [`contrib/codesign.sh`](https://github.com/JuliaLang/julia/blob/8076517c979c703ebed3f1ca5494be2eddfdc30f/contrib/codesign.sh).  Should that also be changed to use the new script without keeping duplicate stuff?

Reply:
Yes, that'd be great.